src/index.js

```diff
@@ -0,0 +1,51 @@
+import Artist from './Artist';
+import Catalog from './Catalog';
+import Network from './Network';
+import PlaylistDynamic from './PlaylistDynamic';
+import Song from './Song';
+import Track from './Track';
+
+import sharedPlaylistParams from './sharedPlaylistParams';
+
+export default class TheEchoNest {
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'class' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).